nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/rest/RestESDocumentsTest.java

```diff
@@ -92,11 +95,34 @@
 
     public static final String TEST_MIME_TYPE = "text/plain";
 
+    /**
+     * @since 11.1
+     */
+    protected static final String PATH_SEPARATOR = "/";
```

Comment:
It's not used as a path separator but as a document parent path when creating documents.

Reply:
But can be used as both without renaming in the future.
wdyt ?

Reply:
It's only used as the Root document path here, so make it `ROOT_PATH`.
Do you really find
```
session.createDocumentModel(PATH_SEPARATOR, COLLECTION_NAME + 1, COLLECTION_TYPE)`
```
meaningful? I don't 🤷‍♂ 

That would be lovely if we can know the future, but we can't, so if there is a need for a `PATH_SEPARATOR` in the future, we will add one. :)